packages/Table/src/TableHead/SortableCell/SortableCell.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+const SortableCell = ({
+  onSort,
+  orderBy,
+  column,
```

Comment:
I would suggest a different name for column, is too similar to "columns". Perhaps "selectedColumn" or sth like that?